test/test_managercli.py

```diff
@@ -1560,13 +1567,14 @@ def test_he_ssl_error(self):
             self.assertEqual(e.code, os.EX_SOFTWARE)
 
     def test_he_ssl_wrong_host(self):
-        e = SSL.Checker.WrongHost("expectedHost.example.com",
-                                   "actualHost.example.com",
-                                   "subjectAltName")
-        try:
-            managercli.handle_exception("huh", e)
-        except SystemExit as e:
-            self.assertEqual(e.code, os.EX_SOFTWARE)
+        if six.PY2:
+            e = SSL.Checker.WrongHost("expectedHost.example.com",
+                                       "actualHost.example.com",
+                                       "subjectAltName")
+            try:
+                managercli.handle_exception("huh", e)
+            except SystemExit as e:
+                self.assertEqual(e.code, os.EX_SOFTWARE)
```

Comment:
What about Python 3?

Reply:
This is a test of a M2Crypto-specific interface. I'll change it to `raise SkipTest` to be more clear.